Weather/WeatherForecast/harvest/spain_weather_forecast_harvest.py

```diff
@@ -169,16 +161,8 @@ def decode_weather_type(weather_type):
         'nuboso con tormenta y lluvia escasa': 'cloudy, thunder, lightRainShower',
         'muy nuboso con tormenta y lluvia escasa': 'veryCloudy, thunder, lightRainShower',
         'cubierto con tormenta y lluvia escasa': 'overcast, thunder, lightRainShower',
-        'despejado noche': 'clearNight'}
-
-    if param in dictionary:
-        out = dictionary[param] + trailing
-    else:
-        out = None
-
-    return out
-
-#########
+        'despejado noche': 'clearNight'}.get(param, None)
+    return out + trailing if out else None
```

Comment:
are you sure you are returning the same?

Reply:
Why?  What am I missing?